L1Trigger/L1THGCal/test/testHGCalBackendStage1ParameterExtractor_cfg.py

```
@@ -62,7 +62,10 @@
 from re import match
 geomXMLcontent = process.XMLIdealGeometryESSource.geomXMLFiles.value()
 hgcal_xml = 'Geometry/HGCalCommonData/data/hgcal/v.*/hgcal.xml'
-geometry_version = "".join(filter(lambda v: match(hgcal_xml,v), geomXMLcontent)).split('/')[4]
+geometry_version = list(filter(lambda v: match(hgcal_xml,v), geomXMLcontent))
+if len(geometry_version) > 1:
+    raise ValueError('More than one xml file could be found for geometry version extraction')
```

In principle, should also raise an exception if the length is 0, otherwise there will be an out-of-bound exception raised in the next line

---

indeed, now fixed